certbot/certbot/_internal/plugins/standalone.py

```diff
@@ -46,7 +45,7 @@ class ServerManager:
     will serve the same URLs!
 
     """
-    def __init__(self, certs: Mapping[bytes, Tuple[crypto.PKey, crypto.X509]],
+    def __init__(self, certs: Mapping[bytes, crypto_util._KeyAndCert],
```

Comment:
using this attribute prefixed with an underscore across a package boundary of ours makes me a little nervous. i worry a future dev will move it and break acme & certbot's compatibility outside of a major version release

can we either:

1. just recreate the `Union[Tuple[crypto.PKey, crypto.X509], Tuple[types.CertificateIssuerPrivateKeyTypes, x509.Certificate],]` here under a different name and then use it or do the same in certbot's standalone_test.py
2. change the name of the variable in `crypto_util` to something like `KeyAndCert` or `KeyCertPair`

Reply:
Went with duplicating it.